packages/account-v2/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "@deriv/account-v2",
+    "version": "2.0.0",
```

Comment:
it should be 1

Reply:
Its actually the v2, but since we cant delete account package, we just renamed this to v2 with updated version number